### ISSUE TITLE
Machines reverting to standard vanilla - 1.10.2 Spigot

### ISSUE THREAD

**Issue by TheSlimeMan** [Kazzababe/TekkitInspired#10]
So, I'm not sure if this is me or what...
Whenever I reload or stop my server (No other plugins other than viaversion and tekkit), All of the machines just become standard blocks.

For example, the filters turn into normal dispensers with a piece of paper in the center slot, pipes are just stained glass, and quarries are just iron blocks.

I have tried deleting the plugin and reinstalling it.

Any ideas?

**Comment by EverNife** [comment]
can reproduce without viaversion ?

**Comment by TheSlimeMan** [comment]
Yes, I tested with and without viaversion. not the issue it seems

**Comment by EverNife** [comment]
can be a problem with the save system...

Console has any logs that indicate errors ?

**Comment by TheSlimeMan** [comment]
Nope. **Edit: How often should every save be? If there is supposed to be something in console about the plugin saving machines, it's not happening on my server.**

**Comment by EverNife** [comment]
i think it saves every "/save-all"

**Comment by Kinoleon** [comment]
same problem here, pipes change to glass and some filters to standard dispenser

I use Spigot 1.11.2 and TekkitInspired 0.8.1

**Comment by Kazzababe** [comment]
Machines save every time the world saves and when the server stops, so I wouldn't be too sure as to why it's not saving.

**Comment by Kinoleon** [comment]
Can you do a version to save it every time the machine is placed?

**Comment by Kazzababe** [comment]
I could if I knew how to handle SQLite asynchronously. I tried it once, but I had issues with the file getting locked. I'm sure I could get it to work, but I'd have to mess around with it more.
